### DISCUSSION TITLE
How to train VOC without pretrained model?

### DISCUSSION THREAD

**Discussion by SeHwanJoo** [rwightman/efficientdet-pytorch#163] [Q&A]
I wanna train from the base.

Thank you

**Comment by rwightman** [comment]
Use the cmd line in the README but don't include `--pretrained` to not use pretrained COCO weights ... if you also do not want to use the pretrained weights for the backbone (the default if you don't use pretrained detection weights), add `--no-pretrained-backbone` ... since VOC has less data than COCO or ImageNet this will likely reduce the quality of the results unless you add augmentation beyond what I have here.
